### ISSUE TITLE
ERROR: PCI subsystem ID mismatch

### ISSUE THREAD

**Issue by GrumpyModeler** [notfromstatefarm/nvflash#19]
I have a Gainward Pegasus 3060 12 GB (LHR) and just about any rom I try I get "ERROR: PCI subsystem ID mismatch".

**Comment by 970user** [comment]
Have you already flashed it, if you are attempting to flash to a 3060 make sure it isn't an 8gb model and also some manufacturers have a different board design so i doesnt work or its the wrong gpu model

**Comment by GrumpyModeler** [comment]
I have another 12 GB 3060 GPU that has the exact same board layout and exact same cooler and fan but branded PNY (non LHR). I wanted to flash that BIOS from the Asus. It's the exact same physically. I'm troubleshooting a fault with it.

EDIT: Wrote Asus first. It's actually PNY.

**Comment by 970user** [comment]
So does the gpu have a fault already, also some gpus don't work with other companies like my asus 970 oc wouldnt work with gigabyte, pny or evga and my 1050 ti didn't work with evga. The fact is it could just be one of those examples where it doesn't work but you can find other bioses. Also LHR vs none LHR have different bioses due to one having a limited hash rate for mining so there could be some kind of board difference there

**Comment by GrumpyModeler** [comment]
It cuts out to black after a couple of seconds of 3D graphics. I wanted to see if there was somehow a problem with the LHR mechanism that caused it by flashing a non LHR bios. VRM still put out voltage after the black screen happens so I expect the core to be borked or some BGA solderings to be bad if it doesn't help.

Flashing the bios of other cards with the same id works. I hoped nvflashk would let me flash any bios I wanted.

**Comment by 970user** [comment]
I don't think it's NVflashk but rather the gpu itself

**Comment by GrumpyModeler** [comment]
I was able to flash a bios from another card with the same ID to it but it's also a LHR BIOS.

**Comment by 970user** [comment]
So what I'm guessing is happening here is that the cards are LHR bios only locked or the ID is different in some way

**Comment by GrumpyModeler** [comment]
Yes. Subsystem ID is the problem. I wanted to report that nvflashk can't bypass the subsystem ID. I assume GPU ID or Board ID were ignoerd since subsystem ID was the only error to show up.
